src/lib/components/BlogLink.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+  export let id: string;
+  export let title: string;
+  export let publishedAt: string;
+  export let tags: string[];
+  export let ogpImg: { url: string } | null;
+
+  function formatDate(dateString: string | number | Date) {
+    const date = new Date(dateString);
+    return date.toLocaleDateString("ja-JP", {
+      year: "numeric",
+      month: "2-digit",
+      day: "2-digit",
+    });
+  }
+</script>
+
+<div>
+  <div class="ogp">
+    <a href={`/blogs/${id}`}>
+      {#if ogpImg}
+        <img src={ogpImg.url} alt={title} />
```

Comment:
```suggestion
        <img
          src={ogpImg.url}
          alt={title}
          height={ogpImg.height}
          width={ogpImg.width}
        />
```

アクセシビリティ的にも [CLS - web.dev ↗](https://web.dev/articles/cls?hl=ja) 的にも, `img` タグには height と width 属性を指定すると良い感じです.  
実は microCMS の画像フィールドには画像の URL の他に, 縦幅と横幅が返される [^1] ので, それを使ってみてはどうでしょう？

[^1]: [メディア管理 - microCMS docs ↗](https://document.microcms.io/manual/media-management#h3ec2b82f0c)